modules/ROOT/pages/hardening-your-mule-installation.adoc

```diff
@@ -4,16 +4,13 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: fine tuning, configuration, security
 
-* <<Hardening Checklist>>
-* <<Hardening in Layers>>
+As distinct from Security, hardening refers to the steps one must perform in order to bring an application from development into production. These are the hoops one must jump through in order to deploy something in the IT space after you complete the development cycle.
```

Comment:
"These are the hoops one must jump through" this phrase might be difficult to translate, it could be changed to something more simple like: "These are elaborate procedures one must go through"